ml_git/ml_git_message.py

```diff
@@ -146,6 +146,7 @@
     'INFO_AMOUNT_SIZE': 'Amount of files: %s',
     'INFO_NOT_INITIALIZED': 'The %s doesn\'t have been initialized.',
     'INFO_NONE_ENTITY_MANAGED': 'You don\'t have any entity being managed.',
+    'INFO_NONE_ENTITY_FILES_MANAGED': 'You don\'t have any files being managed by this entity.',
```

Comment:
The fsck command is applied at the project level and not per entity. Usage: `ml-git datasets fsck
`
```suggestion
    'INFO_NONE_ENTITY_FILES_MANAGED': 'You don\'t have any files being managed in this project.',
```